api/graphql/schema.graphql

```diff
@@ -824,6 +824,7 @@ input PoolCandidateSearchInput {
   processNumber: String @scope(name: "processNumber")
 }
 
```

Comment:
Stray newline

Reply:
Removed in [remove stray line in schema](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11300/commits/d5fff4b35c2f2f8a2955a84dea33f6863f9ad982)